lib/between_meals/knife.rb

```diff
@@ -242,11 +244,9 @@ def write_user_config
       BLOCK
 
       begin
-        unless File.directory?(File.dirname(@pem))
-          Dir.mkdir(File.dirname(@pem), 0o755)
-        end
-      rescue Errno::EEXIST => e
-        @logger.warn("#{File.dirname(@pem)} already exists, not creating: #{e}")
+        Dir.mkdir(File.dirname(@pem), 0o755)
```

Comment:
It seems to me the previous code was more accurate:
```
irb(main):001:0> Dir.mkdir('/etc')
Traceback (most recent call last):
        3: from /usr/bin/irb:11:in `<main>'
        2: from (irb):1
        1: from (irb):1:in `mkdir'
Errno::EEXIST (File exists @ dir_s_mkdir - /etc)
irb(main):002:0> 
```

Reply:
I think it's a matter of style. I was looking at FileUtils.mkdir_p which does this. I've verified this works the same on Linux and Windows. The only messy bit is that we have literally nothing to do in the rescue, so must put `nil` to appease rubocop.